### PR TITLE
i9-add-nuget-attributes

### LINKED ISSUE #9: Add attributes for NuGet deployment

### LINKED ISSUE #11: Adapt appVeyor for NuGet attributes
Relates to #9

### PR DESCRIPTION
**Pull request by TheSonOfDeimos**
Closes #9 
Closes #11

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -7,3 +9,12 @@ test_script:
 
 build_script:
 - dotnet build ./src/ -v m -c Release
+
+build:
+  publish_nuget: true
+
+deploy:
+  provider: NuGet
+  api_key: oy2n3sxah4nvfc4lfphquuynozc37bfujjfhjy3bvhmt64
```
Comment (vkamiansky) [inline appveyor.yml]: Never expose your API Key like that! 
Use the approach from [Composite project](https://github.com/vkamiansky/composite/blob/master/appveyor.yml)
Only publish releases on master tagged with version number.
Reuse secure API key from Composite. We'll publish it under vkamiansky. We might want to remove this one https://www.nuget.org/packages/FlsResults/.

File: src/Fls.Results/Fls.Results.csproj [hunk 2]
```diff
@@ -2,6 +2,21 @@
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
```
Comment (vkamiansky) [inline src/Fls.Results/Fls.Results.csproj]: We might want to lower the netstandard version. We can do better than 2.0. 1.0 would be ideal but we can settle for the lowest version possible.

File: src/Fls.Results/Fls.Results.csproj [hunk 3]
```diff
@@ -2,6 +2,21 @@
...
-  </PropertyGroup>
 
+    <IncludeSymbols>true</IncludeSymbols>
+    <IncludeSource>true</IncludeSource>
+    <PackageId>FlsResults</PackageId>
+    <Title>Project FlsResults</Title>
```
Comment (vkamiansky) [inline src/Fls.Results/Fls.Results.csproj]: Make it 'Fls.Results'.

File: src/Fls.Results/Fls.Results.csproj [hunk 4]
```diff
@@ -2,6 +2,21 @@
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
-  </PropertyGroup>
 
+    <IncludeSymbols>true</IncludeSymbols>
+    <IncludeSource>true</IncludeSource>
+    <PackageId>FlsResults</PackageId>
```
Comment (vkamiansky) [inline src/Fls.Results/Fls.Results.csproj]: Make it 'Fls.Results'

File: src/Fls.Results/Fls.Results.csproj [hunk 5]
```diff
@@ -2,6 +2,21 @@
...
+    <Title>Project FlsResults</Title>
+    <Version>$(APPVEYOR_BUILD_VERSION)</Version>
+    <Authors>FirstLine Net Core Practice Team</Authors>
```
Comment (vkamiansky) [inline src/Fls.Results/Fls.Results.csproj]: FirstLine Software .Net Core Practice Team

File: src/Fls.Results/Fls.Results.csproj [hunk 6]
```diff
@@ -2,6 +2,21 @@
...
+    <Company>FirstLine Software</Company>
+    <license>https://raw.githubusercontent.com/vkamiansky/results/master/LICENSE</license>
+    <PackageProjectUrl>https://github.com/vkamiansky/results</PackageProjectUrl>
+    <PackageTags>study;practice;</PackageTags>
+    <RepositoryUrl>https://github.com/vkamiansky/results.git</RepositoryUrl>
+    <Description>A C# framework for building railroad-oriented code with results.</Description>
+    <Copyright>Copyright (c) 2020, FirstLine Net Core Practice Team</Copyright>
```
Comment (vkamiansky) [inline src/Fls.Results/Fls.Results.csproj]: FirstLine Software .Net Core Practice Team

File: src/Fls.Results/Fls.Results.csproj [hunk 7]
```diff
@@ -2,6 +2,21 @@
...
+    <IncludeSymbols>true</IncludeSymbols>
+    <IncludeSource>true</IncludeSource>
+    <PackageId>FlsResults</PackageId>
+    <Title>Project FlsResults</Title>
+    <Version>$(APPVEYOR_BUILD_VERSION)</Version>
+    <Authors>FirstLine Net Core Practice Team</Authors>
+    <Company>FirstLine Software</Company>
+    <license>https://raw.githubusercontent.com/vkamiansky/results/master/LICENSE</license>
+    <PackageProjectUrl>https://github.com/vkamiansky/results</PackageProjectUrl>
+    <PackageTags>study;practice;</PackageTags>
```
Comment (vkamiansky) [inline src/Fls.Results/Fls.Results.csproj]: Replace these with c#; result; railroad

File: src/Fls.Results/Fls.Results.csproj [hunk 8]
```diff
@@ -2,6 +2,21 @@
...
+    <RepositoryUrl>https://github.com/vkamiansky/results.git</RepositoryUrl>
+    <Description>A C# framework for building railroad-oriented code with results.</Description>
```
Comment (vkamiansky) [inline src/Fls.Results/Fls.Results.csproj:15]: A C# framework for building railroad-oriented code with generic results.

File: src/Fls.Results/Fls.Results.csproj [hunk 9]
```diff
@@ -2,6 +2,21 @@
...
+    <Copyright>Copyright (c) 2020, FirstLine Net Core Practice Team</Copyright>
+
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
```
Comment (vkamiansky) [inline src/Fls.Results/Fls.Results.csproj]: No need for this.

File: src/Fls.Results/Fls.Results.csproj [hunk 10]
```diff
@@ -2,6 +2,21 @@
...
+  </PropertyGroup>
```
Comment (vkamiansky) [inline src/Fls.Results/Fls.Results.csproj:17]: Also add these:
```xml
    <IncludeSymbols>true</IncludeSymbols>
    <IncludeSource>true</IncludeSource>
    <Version>$(APPVEYOR_BUILD_VERSION)</Version>
```

File: README.md [hunk 11]
```diff
@@ -2,4 +2,13 @@
 
 [![Build status](https://ci.appveyor.com/api/projects/status/ol33akfttn7sl176?svg=true)](https://ci.appveyor.com/project/vkamiansky/results)
 
+## Discription
 A C# framework for building railroad-oriented code with results.
+
+## Install from NuGet
+
+```bash
+dotnet add package FlsResults --version 1.0.0
```
Comment (vkamiansky) [inline README.md]: Will add Nuget badge instead when we are ready to release.

**Review by vkamiansky — CHANGES_REQUESTED** [review]
Significant changes in approach are required.

File: src/Fls.Results/Fls.Results.csproj (hunk 8 above)
Comment (TheSonOfDeimos) [reply src/Fls.Results/Fls.Results.csproj:15]: Here is the same text as you've offered

File: src/Fls.Results/Fls.Results.csproj (hunk 10 above)
Comment (TheSonOfDeimos) [reply src/Fls.Results/Fls.Results.csproj:17]: Several lines above, it has been added

File: README.md [hunk 12]
```diff
@@ -2,4 +2,8 @@
 
 [![Build status](https://ci.appveyor.com/api/projects/status/ol33akfttn7sl176?svg=true)](https://ci.appveyor.com/project/vkamiansky/results)
 
+## Discription
```
Comment (vkamiansky) [inline README.md]: Spelling: description.

File: src/Fls.Results/Fls.Results.csproj [hunk 13]
```diff
@@ -1,7 +1,21 @@
...
+    <Title>Fls.Results</Title>
+    <Version>$(APPVEYOR_BUILD_VERSION)</Version>
+    <Authors>FirstLine Software .Net Core Practice Team</Authors>
+    <Company>FirstLine Software</Company>
+    <license>https://raw.githubusercontent.com/vkamiansky/results/master/LICENSE</license>
+    <PackageProjectUrl>https://github.com/vkamiansky/results</PackageProjectUrl>
+    <PackageTags>c#;result;railroad</PackageTags>
+    <RepositoryUrl>https://github.com/vkamiansky/results.git</RepositoryUrl>
+    <Description>A C# framework for building railroad-oriented code with results.</Description>
+    <Copyright>Copyright (c) 2020, FirstLine Net Core Practice Team</Copyright>
```
Comment (vkamiansky) [inline src/Fls.Results/Fls.Results.csproj]: still abides.

File: src/Fls.Results/Fls.Results.csproj [hunk 14]
```diff
@@ -1,7 +1,21 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
-  </PropertyGroup>
+    <TargetFramework>netstandard1.0</TargetFramework>
+
```
Comment (vkamiansky) [inline src/Fls.Results/Fls.Results.csproj]: Format file.

**Review by vkamiansky — CHANGES_REQUESTED** [review]
Minor fixes required.

File: src/Fls.Results/Fls.Results.csproj (hunk 13 above)
Comment (TheSonOfDeimos) [reply src/Fls.Results/Fls.Results.csproj]: I don't understand, what this comment about.

**Review by vkamiansky — APPROVED** [review]

**Pull request merged**